database/__init__.py

```diff
@@ -41,28 +41,22 @@ class Role(db.Model):
         id = db.Column(db.Integer, primary_key=True)
         role = db.Column(db.String(255))
 
-    class Hotel(db.Model):
-        __tablename__ = "hotels"
-        id = db.Column(db.Integer, primary_key=True)
-        name = db.Column(db.String(255))
-        api_endpoint = db.Column(db.String(255))
-        number_of_rooms = db.Column(db.Integer)
-        city = db.Column(db.String(255))
-        address = db.Column(db.String(255))
-        rating = db.Column(db.Float)
-
     class Room(db.Model):
         __tablename__ = "rooms"
         id = db.Column(db.Integer, primary_key=True)
-        hotel = db.Column(db.Integer, db.ForeignKey("hotels.id"))
+        hotel_name = db.Column(db.String(255))
```

Comment:
czemu hotel_name a nie hotel_id? chyba lepiej ForeignKey?

Reply:
Bo zdecydowaliśmy usunąć tabelę z hotelami i zrobić z tego jedną tabelę w access poincie, a listę tworzymy bez wysyłania zapytań do hotelów, a mając tylko id to nie będziemy mieli żadnej informacji co to za hotel więc będzie to zbędne